src/main/java/com/site/news/demo/domain/NewsItem.java

```diff
@@ -0,0 +1,77 @@
+package com.site.news.demo.domain;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.sql.Date;
+
+@Data
```

Comment:
If we use @Data annotation we don't need getters/setters. Please remove getters/or @Data usage